libs/cardano-ledger-conformance/test/Test/Cardano/Ledger/Conformance/Imp/Ratify.hs

```diff
@@ -152,8 +153,10 @@ spec = withImpInit @(LedgerSpec Conway) $ describe "RATIFY" $ modifyImpInitProtV
     logString "Agda res:"
     logToExpr agdaRes
     logString "Extra information:"
+    globals <- use impGlobalsL
```

Comment:
So is it here that before we were incorrectly using a different globals instead of the one from imp? Just curious which one, i can't find figure out from the the PR what removes the usage of the incorrect globals :thinking: